recipes/poap/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "POAP" %}
+{% set version = "0.1.26" %}
+{% set sha256 = "05d34b8075faf4bc41e1a169151f545ef3e45cfa62ec31f815fa3c06ee2dd4fb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - poap
+    - poap.test
+
+  source_files:
+    - poap/test/*
+
+  commands:
+    - cd poap
+    - cd test
+    - python test_strategies.py
+    - python test_input_strategy.py
+    - python test_coroutine_strategy.py
+    - python test_fixed_strategy.py
+    - python test_maxterm_strategy.py
+    # - python test_process_worker.py  # Disabled because it depends on a compiled `dummy_sim`
+    - python test_simteam_controller.py
+    - python test_thread_controller.py
+    - python test_thread_strategy.py
+    - python test_tcpserve.py
+
+about:
+  home: https://github.com/dbindel/POAP
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Python Optimization Asynchronous Plumbing.'
+  description: |
+    POAP provides an event-driven framework for building and combining
+    asynchronous optimization strategies.
+
+    The basic ingredients are a controller capable of asking workers to run
+    function evaluations and a strategy for choosing where to sample. The
+    strategies send the controller proposed actions, which the controller then
+    accepts or rejects; the controller, in turn, informs the strategies of
+    relevant events through callback functions.
+  dev_url: https://github.com/dbindel/POAP
+
+extra:
+  recipe-maintainers:
+    - tadeu
+    - dbindel
```

Comment:
@dbindel, are you ok being added as a maintainer for this recipe here?

Reply:
@dbindel are you ok with being a maintainer?

Reply:
Since there was no answer, I've removed